### DISCUSSION TITLE
Custom URLs for new tabs

### DISCUSSION THREAD

**Discussion by pwongy** [Floorp-Projects/Floorp#1284] [Feature Request]
It would be great to be able to set new tabs to open a custom URL in Settings > Home. There is already a way to do this for Homepage and new windows, but for new tabs we currently need to use an extension like New Tab Override.

**Comment by CutterKnife** [comment]
Enter your custom URL in `floorp.newtab.overrides.newtaburl` and restart your Floorp.

**Reply by pwongy** [reply]
Oh amazing, I didn't know there was a flag for this - works perfectly! 🙏

Are there any plans to build this into the GUI settings page?

**Reply by CutterKnife** [reply]
Floorp has some about:config-only settings, but they have no plans to add them to about:preferences at this time.

**Reply by pwongy** [reply]
Noted - thank you!
